tools/lsp/preview/native.rs

```diff
@@ -205,8 +205,9 @@ fn show_document_request_from_element_callback(
 ) -> Option<lsp_types::ShowDocumentParams> {
     use lsp_types::{Position, Range, ShowDocumentParams, Url};
 
-    let start_pos = Position::new(start_line - 1, start_column);
-    // let end_pos = Position::new(end_line - 1, end_column);
+    let start_pos =
+        Position::new(u32::saturating_sub(start_line, 1), u32::saturating_sub(start_column, 1));
```

Comment:
I usually write it with this style
```suggestion
        Position::new(start_line.saturating_sub(1), start_column.saturating_sub(1));
```
but feel free to ignore